src/platform/cannonlake/include/platform/memory.h

```diff
@@ -297,10 +296,9 @@
 /* Stack configuration */
 #define SOF_STACK_SIZE		ARCH_STACK_SIZE
 #define SOF_STACK_TOTAL_SIZE	ARCH_STACK_TOTAL_SIZE
-/* SOF_STACK_OFFSET defines how much memory can be power gated */
-#define SOF_STACK_OFFSET	0x150000
+
 /* SOF_STACK_BASE is moved from end of physical memory by offset */
```

Comment:
This comment should be also removed (or changed) if SOF_STACK_OFFSET is not used anymore